client/tailwind.config.js

```diff
@@ -2,7 +2,22 @@
 module.exports = {
   content: ["./**/src/**/*.{tsx, ts}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'gray': '#D9D9D9',
+        'pink': '#FF2480',
+        'getStarted': '#B7194A',
+        'learnMore': '#B7194A',
+        'divider': "ADADAD",
+        'start': '#B7194A'
```

Comment:
This is not how extended color palette should be used (and class naming should be kabab-case)